lib/honeycomb/client.rb

```diff
@@ -69,8 +61,8 @@ def start_span(name:, serialized_trace: nil, **fields)
       begin
         yield current_span
       rescue StandardError => e
-        current_span.add_field("error", e.class.name)
-        current_span.add_field("error_detail", e.message)
+        add_exception_data(current_span, e)
```

Comment:
I see what you did here ... and I like it.

Reply:
You can, once again, think Rubocop for making me multi-line these function calls them yelling at me about the method being too long -.-

Reply:
And you came up with an elegant solution to reconcile the "you should call this exception `e`" cop fighting with our desire for a longer variable name so that the indentation of the method chaining doesn't make our eyes itch. Thank you for the nagging, Rubocop.